crates/task-impls/src/view_sync.rs

```diff
@@ -730,11 +705,6 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>, A: ConsensusApi<TYPES, I> +
                     return (Some(HotShotTaskCompleted::ShutDown), self);
                 }
 
-                // Ignore if the certificate is for an already seen phase
-                if last_seen_certificate <= self.phase {
-                    return (None, self);
-                }
-
```

Comment:
I'm not sure we should delete this?  I think it is possible that a node gets a Commit cert, and then also somehow gets the PreCommit cert afterwards, say because of network delay.  In this case I think it would incorrectly reset itself to be in the PreCommit phase, which isn't what we want.

Reply:
The phase doesn't really matter that I think.  In this code it only matters if we get to finalize since we don't use the phase anything on it's own.  I think we could probably get rid of self.phase but I was trying to get something done quickly